### ISSUE TITLE
UniqueID

### ISSUE THREAD

**Issue by KarellenX** [pkscout/metadata.tvshows.themoviedb.org.python#6]
Could you also add scraping of IMDB ID and TVDB ID to be used as UniqueID's for both **TV Show** and **Episodes**.

These ID's, especially the IMDB ID, can be used by other add-ons.

FYI. While testing be aware of this issue, which Montellese has confirmed, or it can drive you a bit mad as it did me ;)... https://github.com/xbmc/xbmc/issues/17444

![uniqueid](https://user-images.githubusercontent.com/44626605/93050814-dfebd480-f6a6-11ea-9aa4-eaec63f6058d.JPG)

**Comment by pkscout** [comment]
added with 058ad7b790a53a7fdda34574182e83d616ef994f. Because Freebase and TVRage are defunct, the scraper only adds tmdb, tvdb, and imdb ids.

**Comment by KarellenX** [comment]
All working well. Strangely, I scraped three complete shows and I did not encounter the bug I linked above.

Thanks!!

**Comment by KarellenX** [comment]
Just noticed an issue with UniqueID's

If the UniqueID's for IMDB and TVDB are empty at TheMovieDB (which happens often) then the scraper is forcing a non-UniqueID as shown in the example below.

```xml    
<uniqueid type="imdb">None</uniqueid>
<uniqueid type="tmdb" default="true">1778196</uniqueid>
<uniqueid type="tvdb">None</uniqueid>
```

If a uniqueid is not available, then (in the above example) imdb and tvdb entries should not be made.

![uniqueid](https://user-images.githubusercontent.com/44626605/93692948-d9dd7400-fb3c-11ea-8363-f3f810d92c6f.JPG)

**Comment by pkscout** [comment]
fixed with 601c7ebadece107ac9891440a04d00c74aebd24e

**Comment by KarellenX** [comment]
Bug fixed.
